apps/daffio/src/app/docs/components/doc-viewer/doc-viewer.component.html

```diff
@@ -4,29 +4,31 @@
 </button>
 <div class="daffio-doc-viewer__grid">
   <div class="daffio-doc-viewer__content">
-    @if (doc.breadcrumbs?.length > 0) {
-      <nav aria-label="Breadcrumb">
-        <ol daff-breadcrumb>
-          @for (breadcrumb of doc.breadcrumbs; track $index) {
-            <li daffBreadcrumbItem [active]="$last">
-              @if ($last) {
-                {{breadcrumb.label}}
-              } @else {
-                <a [routerLink]="breadcrumb.path">{{breadcrumb.label}}</a>
-              }
-            </li>
-          }
-        </ol>
-      </nav>
-    }
-
-    @if (isApiPackage) {
-      <daffio-api-package [doc]="doc"></daffio-api-package>
-    } @else {
-      <daff-article
-        [innerHtml]="getInnerHtml(doc)">
-      </daff-article>
-    }
+    <daff-article>
+      @if (doc.breadcrumbs?.length > 0) {
+        <nav aria-label="Breadcrumb">
+          <ol daff-breadcrumb>
+            @for (breadcrumb of doc.breadcrumbs; track $index) {
+              <li daffBreadcrumbItem [active]="$last">
+                @if ($last) {
+                  {{breadcrumb.label}}
+                } @else {
+                  <a [routerLink]="breadcrumb.path">{{breadcrumb.label}}</a>
+                }
+              </li>
+            }
+          </ol>
+        </nav>
+      }
+  
+      <ng-content>
```

Comment:
Why is the `<ng-content>` wrapping the api package and the div instead of being sibling like this, assuming the `<ng-content>` is used to slot in the tabs component?

```
<ng-content></ng-content>
@if (isApiPackage) {
  <daffio-api-package [doc]="doc"></daffio-api-package>
} @else {
  <div [innerHtml]="getInnerHtml(doc)"></div>
}
```

Reply:
Is this comment in issue #3390 related to this?

> I'm going to change the doc viewer component to be more flexible beyond just dumbly rendering whatever is in the content field. We will need that flexibility to implement https://github.com/graycoreio/daffodil/pull/3377#issuecomment-2566693685

Reply:
yes, this will be changed completely, I will open the PR soon